.github/ISSUE_TEMPLATE/kubernetes-onboarding-template.md

```diff
@@ -71,6 +71,7 @@ _Requesting team; Infrastructure Team can assist_
 - [ ] **Kubernetes manifest in YAML, Jsonnet, or Helm Chart** in `vsp-infra-application-manifests` | [Example](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/tree/main/apps) | [Docs](https://argo-cd.readthedocs.io/en/stable/)  
 Note: must be compatible with ArgoCD  
 _Requesting team; Infrastructure Team can assist_  
+- [ ] **Kubernetes detect and update application** in `vsp-infra-applications-manifests` | [Environments](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/tree/main/apps/vsp-operations/argocd-apps)
```

Comment:
Let's put who is responsible underneath. 
like, _Requesting team; Infrastructure Team can assist_